orbit-db-cli/src/services/start-ipfs.js

```diff
@@ -0,0 +1,58 @@
+import { ipfsConfig, repoPath } from './config';
+
+// defaultRepoPath: string
+const defaultRepoPath = repoPath || './ipfs/';
+
+const getLoadCodec = async () => {
+  const dagCbor = await import("@ipld/dag-cbor");
+  const dagPb = await import("@ipld/dag-pb");
+  const raw = await import("multiformats/codecs/raw");
+
+  // nameOrCode: string | number
+  const loadCodec = (nameOrCode) => {
+    // codecs: Record<string, any>
+    const codecs = {
```

Comment:
why do you put a comment instead of just typing it?
```suggestion
    const codecs: Record<string, any> = {
```

Reply:
Le fichier est en JavaScript